config/numpy.sh

```diff
@@ -11,15 +11,19 @@ echo "[lapack]" >> site.cfg;
 echo "libraries = flexiblas" >> site.cfg;
 echo "[blas]" >> site.cfg;
 echo "libraries = flexiblas" >> site.cfg;'
-MODULE_BUILD_DEPS="arch/sse3 flexiblas/3.0.4"
+MODULE_BUILD_DEPS="flexiblas"
+if [[ $EBVERSIONGENTOO != 2023 ]]; then
+	MODULE_BUILD_DEPS="arch/sse3 $MODULE_BUILD_DEPS"
```

Comment:
don't we need to load arch/avx2 for the 2023 environment ?

Reply:
It is the default in the current environment right (avx2)

Reply:
yes, unless you set it explicitly in RSNT_ARCH:
`module load arch/${ARCH_TO_LOAD:-avx2}`
in `build_wheel.sh`

Reply:
That was also the case of `sse3` in the 2020 environment, isn't it ? I thought this specific example in `numpy` was an override to *always* use `sse3` rather than rely on `RSNT_ARCH` ?

Reply:
Indeed, it is was override since numpy does runtime detection of the architecture. But in 2023 the default is avx2. Perhaps, in order to ensure that this wheel is always _generic_ we should set to avx2 by default as it was done for sse3

Reply:
done